api/src/cli/commands/schema/snapshot.ts

```diff
@@ -8,13 +8,11 @@ import { dump as toYaml } from 'js-yaml';
 
 export async function snapshot(
 	snapshotPath?: string,
-	options?: { yes: boolean; format: 'json' | 'yaml' }
+	options?: { yes: boolean; format: 'json' | 'yaml'; split: 'yes' | 'no' }
 ): Promise<void> {
 	const database = getDatabase();
 
-	try {
-		const snapshot = await getSnapshot({ database });
-
+	async function saveSnapshot(snapshot: any, suffix = '') {
```

Comment:
thoughts on making suffix an optional param (`suffix?: string`) rather than default to `''`?